tests/system_tests/endpoint_tests/migrations/20240115223956_add_sample_posts.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+-- +goose StatementBegin
+INSERT INTO posts(title, content) VALUES(
```

Comment:
This is also probably not meant to be here: we already have some migrations that add the posts!